lib/hoe/publish.rb

```diff
@@ -100,7 +100,7 @@ def initialize_publish
   # Declare a dependency on rdoc, IF NEEDED.
 
   def activate_publish_deps
-    dependency "rdoc", "~> 4.0", :developer if need_rdoc
+    dependency "rdoc", ">= 4.0", :developer if need_rdoc
```

Comment:
How would you handle this scenario then where you need to work with rdoc "~> 4.0" and rdoc "~> 5.0"?